molecule/default/requirements.yml

```diff
@@ -1,5 +1,16 @@
 ---
+<<<<<<< HEAD
 - src: https://github.com/cisagov/ansible-role-cyhy-core
   name: cyhy_core
 - src: https://github.com/cisagov/ansible-role-ncats-webd
   name: ncats_webd
+=======
+>>>>>>> 5fe7ba99c2b2eb5be18c2d0a5466949a714591a8
```

Comment:
This file still has merge markers in it.

Reply:
This got Lineage clobbered while still in review because of the triple dash PR upstream.

Reply:
Resolved (and tests finally finished after a couple restart attempts) in f053158